src/mscorlib/src/System/Tuple.cs

```diff
@@ -118,7 +118,7 @@ public Tuple(T1 item1)
 
         public override Boolean Equals(Object obj)
         {
-            return ((IStructuralEquatable)this).Equals(obj, EqualityComparer<Object>.Default);
+            return ((IStructuralEquatable)this).Equals(obj, null);
```

Comment:
@jcouv Because there is a chance that the comparer is never used, and because it's not DRY. I asked the same thing when I did ConcurrentDictionary and I'm following @hughbe's [answer](https://github.com/dotnet/corefx/pull/18441#issuecomment-294334192).